Misc spec text gaps

ptomato opened the pull request.

Ms2ger commented on spec/calendar.html:
```diff
@@ -59,9 +59,27 @@ <h1>CalendarFields ( _calendar_, _fieldNames_ )</h1>
+        1. Let _options_ be ! ObjectCreate(%Object.prototype%).
+        1. Perform ! CreateDataPropertyOrThrow(_options_, *"overflow"*, _overflow_).
+        1. Let _dateAdd_ be ? Get(_calendar_, *"dateAdd"*).
+        1. Return ? Call(_dateAdd_ , _calendar_, « _date_, _duration_, _options_, _constructor_ »).
```
```suggestion
        1. Return ? Call(_dateAdd_, _calendar_, « _date_, _duration_, _options_, _constructor_ »).
```

Ms2ger replied:
Should this check the return value?

Ms2ger commented on spec/plainyearmonth.html:
```diff
@@ -249,10 +260,13 @@ <h1>Temporal.PlainYearMonth.prototype.add ( _temporalDurationLike_ [ , _options_
         1. If _sign_ &lt; 0, then
-          1. Let _day_ be <mark>TODO: get via calendar.daysInMonth</mark> ! DaysInMonth(_yearMonth_.[[ISOYear]], _yearMonth_.[[ISOMonth]]).
+          1. Let _daysInMonth_ be ? Get(_calendar_, *"daysInMonth"*).
+          1. Let _day_ be ? Call(_daysInMonth_, _calendar_, « _yearMonth »).
```
todo: ToInteger?

ptomato replied:
I will address this in a follow up, since it seems like actually all these methods should be called by abstract operations that then call ToInteger on the result.

Ms2ger commented on spec/plainyearmonth.html:
```diff
@@ -249,10 +260,13 @@ <h1>Temporal.PlainYearMonth.prototype.add ( _temporalDurationLike_ [ , _options_
         1. If _sign_ &lt; 0, then
-          1. Let _day_ be <mark>TODO: get via calendar.daysInMonth</mark> ! DaysInMonth(_yearMonth_.[[ISOYear]], _yearMonth_.[[ISOMonth]]).
+          1. Let _daysInMonth_ be ? Get(_calendar_, *"daysInMonth"*).
+          1. Let _day_ be ? Call(_daysInMonth_, _calendar_, « _yearMonth »).
```
Missing underscore on "yearMonth". Also in subtract() below.

ptomato commented on spec/abstractops.html:
```diff
@@ -1236,6 +1238,7 @@ <h1>ParseTemporalMonthDayString ( _isoString_ )</h1>
       1. Else,
         1. Set _day_ to ! ToInteger(_day_).
       1. Return the new Record {
+        [[Year]]: _year_,
```
I think those are left over from old code. I've removed them from the polyfill.

ptomato commented:
I've addressed the comments and will follow up on the ToInteger one, merging this one now. Please let me know if there emerges anything else I should follow up on.
